src/main/resources/resources/logisim/strings/gui/gui_de.properties

```diff
@@ -624,45 +624,45 @@ windowGridColorsReset = Grid Farbe auf den Logisim standart zurücksetzen
 #
 # prefs/HotkeyOptions.java
 #
-# ==> hotkeyOptMenuKeyHeader =
-# ==> hotkeyOptNormalKeyHeader =
-# ==> hotkeyOptOrientDesc =
+hotkeyOptMenuKeyHeader = Menü Shortcuts die %s beinhalten müssen:
+hotkeyOptNormalKeyHeader = Situationsabhängige Shortcuts die %s nicht benötigen:
+hotkeyOptOrientDesc = Componenten ausrichten
 # ==> hotkeyOptHelp =
-# ==> hotkeyOptTitle =
-# ==> hotkeyOptResetBtn =
-# ==> hotkeySimAutoPropagate =
-# ==> hotkeySimReset =
-# ==> hotkeySimStep =
-# ==> hotkeySimTickHalf =
-# ==> hotkeySimTickFull =
-# ==> hotkeySimTickEnabled =
-# ==> hotkeyEditUndo =
-# ==> hotkeyEditRedo =
-# ==> hotkeyWindowClose =
-# ==> hotkeyWindowMinimize =
-# ==> hotkeyFileExport =
-# ==> hotkeyFilePrint =
-# ==> hotkeyErrMeta =
-# ==> hotkeyErrConflict =
-# ==> hotkeyDirNorth =
-# ==> hotkeyDirSouth =
-# ==> hotkeyDirEast =
-# ==> hotkeyDirWest =
-# ==> hotkeyEditMenuDuplicate =
-# ==> hotkeyEditToolDuplicate =
-# ==> hotkeyProjMoveUp =
-# ==> hotkeyProjMoveDown =
-# ==> hotkeyAutoLabelOpen =
-# ==> hotkeyAutoLabelToggle =
-# ==> hotkeyAutoLabelView =
-# ==> hotkeyAutoLabelHide =
+hotkeyOptTitle = Shortcuts Einstellungen
+hotkeyOptResetBtn = Auf Standardeinstellungen zurücksetzen
+hotkeySimAutoPropagate = Automatisch weiterschalten
+hotkeySimReset = Simulation zurücksetzen
+hotkeySimStep = Simulation weiterschalten
+hotkeySimTickHalf = Einen halben Zyklus weiterschalten
+hotkeySimTickFull = Einen vollen Zyklus weiterschalten
+hotkeySimTickEnabled = Simulator kontinuierlich betreiben
+hotkeyEditUndo = Rückgängigmachen
+hotkeyEditRedo = Wiederherstellen
+hotkeyWindowClose = Fenster schließen
+hotkeyWindowMinimize = Fenster minimieren
+hotkeyFileExport = Datei exportieren
+hotkeyFilePrint = Design drucken
+hotkeyErrMeta = Der Shortcut muss die %s Taste enthalten. 
+hotkeyErrConflict = Überschneidung mit: %s.
+hotkeyDirNorth = Norden
+hotkeyDirSouth = Süden
+hotkeyDirEast = Osten
+hotkeyDirWest = Westen
+hotkeyEditMenuDuplicate = Duplizieren
+hotkeyEditToolDuplicate = Auswahl duplizieren
+hotkeyProjMoveUp = Schaltung nach oben
+hotkeyProjMoveDown = Schaltung nach unten
+hotkeyAutoLabelOpen = Label bearbeiten
+hotkeyAutoLabelToggle = Label sichtbarkei umschalten
+hotkeyAutoLabelView = Label anzeigen
+hotkeyAutoLabelHide = Label ausblenden
 # ==> hotkeyAutoLabelSelfNumberedStop =
```

Comment:
I could not think of a good translation for "End Self Numbered Function".

Reply:
I took some time today to actually understand the context of AutoLabelSelfNumberStop. Here is how you can observe its use.

Open a new document. Select a component with a label: I picked a register. Drag one out toward the canvas but do not drop it. Press the lower case L. A popup allows you to enter a label. Pick a label ending in a number: I picked "R1". Now drop the component. It should show with the selected label. Now drag another one onto the canvas and drop it. It should show with the next label, e.g. "R2". Each new one increments the number.

Now suppose you wish to drag one out and not label it. While dragging, press lower case A. That is the AutoLabelSelfNumberStop issue. When you drop it, it is not labeled.

@juli-99 and @MarcinOrlowski: I propose "Stop auto-numbering labels" as a better description in English. But @MarcinOrlowski is correct that getting the context for translation in a short description is pretty difficult. This panel is not the documentation for how to use the key. It allows someone that knows how to use the key to change the key to use.

Note that the Label open action (L) works as described above if using it while dragging out the component. But it also works to apply a label on a selected component that is already placed, but does not start auto-numbering in that case. Many of these keys can also be used on selected items already placed on the canvas.

Reply:
keine automatische Nummerierung